src/libraries/Native/build-native.sh

```diff
@@ -129,9 +129,9 @@ build_native()
     fi
 
     # Regenerate the CMake solution
-    commonDir="$__rootRepo/eng/common"
+    commonDir="$__rootRepo/eng/native"
```

Comment:
Nit: this variable name would now make more sense like: `engNativeDir`

Reply:
Renamed to `engNativeDir`, also update CMAKE variable name to `CLR_ENG_NATIVE_DIR`.